redeem/PathPlanner.py

```diff
@@ -75,11 +75,11 @@ def __init__(self, printer, pru_firmware):
         self.prev.set_prev(None)
 
         if pru_firmware:
-            self.__init_path_planner()
+            self._init_path_planner()
         else:
             self.native_planner = None
 
-    def __init_path_planner(self):
+    def _init_path_planner(self):
```

Comment:
Just for  my edification - is this actually necessary or is it just the python-stylistic equivalent of switching from "private" to "protected" ?

Reply:
since double underscore does name mangling, it makes it difficult to patch the object with a mock.